### PR TITLE
Add --version option

### PR DESCRIPTION
**Pull request by melissalinkert**
The Gradle build will generate a ```version.properties``` file that contains the bioformats2raw and Bio-Formats version numbers.  Running ```bioformats2raw --version``` will print both values and exit.

See #49

### CODE REVIEW

**Comment by chris-allan** [comment]
My strong preference here would be to use the JAR implementation version as we have with the microservices and stitcher:

 * https://github.com/glencoesoftware/omero-ms-image-region/blob/master/build.gradle#L93
 * https://github.com/glencoesoftware/omero-ms-image-region/blob/master/src/main/java/com/glencoesoftware/omero/ms/image/region/ImageRegionMicroserviceVerticle.java#L266
 * https://github.com/glencoesoftware/sticher/blob/master/src/main/java/com/glencoesoftware/stitcher/Main.java#L34

Is there also a reason why we wouldn't use the `FormatTools` constants in the stitcher way as well?

File: build.gradle [hunk 1]
```diff
@@ -26,7 +26,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'ome:formats-gpl:6.4.0'
+    implementation "ome:formats-gpl:6.4.0"
```
Comment (chris-allan) [inline build.gradle:29]: Was there a reason for this?

Comment (melissalinkert) [reply build.gradle]: Not any more; changed back in 7106854.

**Comment by melissalinkert** [comment]
7106854 writes the version numbers to stdout instead of using slf4j, which should look better.  If you prefer to see those logged with slf4j, then we either need to change the logging configuration overall (not ideal), or override the default configuration when ```--version``` is used.

**Comment by chris-allan** [comment]
@melissalinkert: I'm fine with either. STDOUT works for me if you prefer.

**Comment by melissalinkert** [comment]
stdout is completely fine with me.

**Pull request merged**